content/riak/ts/1.4.0/using/querying/select/arithmetic-operations.md

```diff
@@ -11,9 +11,7 @@ project: "riak_ts"
 project_version: "1.4.0"
 toc: true
 version_history:
-  present_from: "1.4.0+"
-  moved:
-    - ["1.1.0+", "using/arithmetic-operations"]
+  in: "1.4.0+"
```

Comment:
Should also have a `locations` with 1.1.0-1.3.1 being `using/arithmetic-operations/` and 1.4.0 being `using/querying/select/arithmetic-operations/`?